target/linux/mediatek/image/filogic.mk

```diff
@@ -141,6 +140,20 @@ define Device/mediatek_mt7986b-rfb
 endef
 TARGET_DEVICES += mediatek_mt7986b-rfb
 
+define Device/mercusys_mr90x-v1
+  DEVICE_VENDOR := Mercusys
+  DEVICE_MODEL := MR90X v1
+  DEVICE_DTS := mt7986a-mercusys-mr90x-v1
```

Comment:
The filename should be mt7986b here.

Reply:
Ah, dts filename... Renamed, `filogic.mk` was updated in https://github.com/openwrt/openwrt/commit/f3a108957fb0991c896fda5e70d4af3bbfebc7c6